index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+    	<meta charset="utf-8">
+    	<meta name="description" content="Your description goes here">
+    	<meta name="keywords" content="one, two, three">
+
+		<title>Relaxer</title>
+
+		<!-- external CSS link -->
+		<link rel="stylesheet" type="text/css" href="https://www.google.com/fonts#UsePlace:use/Collection:Open+Sans">
+		<link rel="stylesheet" href="css/reset.css">
+		<link rel="stylesheet" href="css/style.css">
+
+	</head>
+	<body>
+		<div id="container">
+			<header>
+				<h1>Relaxer</h1>
+				<nav>				
+					<a href="about.html">About</a>
+					<a href="faq.html">FAQ</a>		
+					<a href="team.html">Team</a>			
+					<a href="contactus.html">Contact us</a>	
+					<a href="blog.html">Blog</a>			
+				</nav>
+			</header>
+			<section class="side">
+				<h2>
+				Categories
+				</h2>
+				<div></div>
+				<p>Success stories <br>Stats<br>How tos<br>Best of<br>Business tips</p>
+				<h2>
+				About Relaxr
+				</h2>
+				<div></div>
+				<p>Lorem ipsum sed justo wel sapien varius auctor. fusce porttitor, sapein sed <p class="bold">Learn more</p></p>
+				<img class="imgside" src="./images/ad-unit.png" alt="add">
+			</section>
+			<section class="bodycopy">
+				<h2>How I implemented Relaxr in 2 weeks and changed my life.</h2>
+
+				<div></div>
+				<img src="./images/blog_photo1.jpg" alt="blog">
+				<p><strong>L</strong>orem ipsum dolor sit amet, consectetur adipiscing elit. Ut pellentesque ut quam varius accumsan. Integer ac libero nulla. Integer vel lectus sed orci ullamcorper efficitur a et nisl. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin tristique sit amet diam nec mattis. Vivamus quis pellentesque lacus.</p>
+				<h3> read more ></h3>
+				<div></div>
+				<hr>
+
+			</section>
+						<section class="bodycopy">
+				<h2>How I implemented Relaxr in 2 weeks and changed my life.</h2>
+
+				<div></div>
+				<img src="./images/blog_photo2.jpg" alt="blog">
+				<p><strong>L</strong>orem ipsum dolor sit amet, consectetur adipiscing elit. Ut pellentesque ut quam varius accumsan. Integer ac libero nulla. Integer vel lectus sed orci ullamcorper efficitur a et nisl. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin tristique sit amet diam nec mattis. Vivamus quis pellentesque lacus.</p>
```

Comment:
The rubrics mentioned doing drop caps with pseudo classes.

That is, in css
```
p:first-letter {
    font-size: 50px;
}
```

Notice the ```:first-letter``` this is a pseudo effect on ```<p>``` tags. Similar to ```:hover```